packages/oas/test/lib/openapi-to-json-schema.test.ts

```diff
@@ -908,13 +908,13 @@ describe('`description` support', () => {
           type: 'string',
           enum: ['CP', 'NON_CP'],
           default: 'NON_CP',
-          description: 'Enumed property with a default: `NON_CP`\n\nDefault: `NON_CP`',
+          description: 'Enumed property with a default: `NON_CP`\n',
```

Comment:
where's that `\n` coming from now?

Reply:
it's hardcoded here: https://github.com/readmeio/oas/blob/74fd996a30ab62103335d8342949a2086f3db191/packages/oas/test/lib/openapi-to-json-schema.test.ts#L881

i'm guessing it was there to test the whitespace handling?

Reply:
huh yeah maybe